utils/utils.go

```diff
@@ -0,0 +1,63 @@
+package utils
```

Comment:
I don't agree with having a package named "utils". This is for sure not best practice. Why are those functions in this package and not others? Why are they public